### ISSUE TITLE
App icons in "Jump start your desktop" banner inaccessible

### ISSUE THREAD

**Issue by cassidyjames** [ubuntu/app-center#1594]
The app icons in the featured banner on the home page of App Center are not accessible to Ubuntu's built-in screen reader; it just reads, "Image" and does not include the app name. Further, there is not keyboard focus indication showing which app icon is focused, making it difficult to navigate with the keyboard.
